game-core/src/test/java/org/triplea/lobby/common/GameDescriptionTest.java

```diff
@@ -47,4 +49,22 @@ void isBotShouldReturnFalseWhen() {
                   assertThat(shouldNotBeBot.toString(), shouldNotBeBot.isBot(), is(false)));
     }
   }
+
+  @Nested
+  final class FormatBotStartTimeTest {
+    @Test
+    void shouldNotThrowExceptionWhenStartedDateIsNull() {
+      final GameDescription gameDescription = GameDescription.builder().build();
+
+      assertDoesNotThrow(gameDescription::getFormattedBotStartTime);
+    }
+
+    @Test
+    void shouldNotThrowExceptionWhenStartedDateIsNotNull() {
+      final GameDescription gameDescription =
+          GameDescription.builder().startDateTime(Instant.now()).build();
+
+      assertDoesNotThrow(gameDescription::getFormattedBotStartTime);
```

Comment:
Thinking about this, this test is kinda silly, but I guess the reasoning here is that the time differes depending on the region, so we can't really reliably test different locales.

Reply:
Funny enough, did get a NPE when formatting a null date and that did need to be fixed. Sometimes verifying you get no error is a reasonable minimal level of testing, better than nothing. I'm pretty sure we probably could set the locale for the test and make it more deterministic; I suspect there the test could be made stronger/better.